airflow/models/__init__.py

```diff
@@ -59,12 +59,6 @@ def import_all_models():
     for name in __lazy_imports:
         __getattr__(name)
 
-    import airflow.jobs.backfill_job
```

Comment:
Not needed any more - theose are not polymorphic an do not need to be registered at init.